### DISCUSSION TITLE
scrollback configuration

### DISCUSSION THREAD

**Discussion by ronildo** [Eugeny/tabby#5192] [General]
Is there a configuration for scrollback? I couldn't find anything about it.

![Image 12-16-21 at 3 37 PM](https://user-images.githubusercontent.com/622159/146445601-547dbf28-0d87-4a3c-82c6-c1774033abc8.jpg)

**Comment by Eugeny** [comment]
It's controlled by the `terminal.scrollbackLines` config option, which is not exposed in the GUI.

**Reply by ronildo** [reply]
How can I change my local app to update this configuration?
Sorry but I tried a few ways and wasn't able to increase the default value.
